lib/game/errors.go

```diff
@@ -0,0 +1,9 @@
+package game
+
+import (
+	"errors"
+)
+
+var (
+	errInvalidLength = errors.New("invalid length")
```

Comment:
Can we make a slightly more descriptive error message? Something along the lines of "given array does not match universe size" or similar

Reply:
Good idea. Done.